src/bot.js

```diff
@@ -254,6 +254,23 @@ client.on(Events.MessageCreate, async message => {
 		let userInput = message.content
 			.replace(new RegExp("^\s*" + myMention.source, ""), "").trim();
 
+		// Process text files if attached
+		if (message.attachments.size > 0) {
+			const textAttachments = Array.from(message.attachments, ([, value]) => value).filter(att => att.contentType.startsWith("text"));
+			if (textAttachments.length > 0) {
+				userInput = `${message.content}`;
+				try {
+					await Promise.all(textAttachments.map(async (att, i) => {
+						const response = await axios.get(att.url);
+						userInput += `\n${i + 1}. File - ${att.name}:\n${response.data}`;
+					}));
+				} catch (error) {
+					log(LogLevel.Error, `Failed to download text files: ${error}`);
+					return; // Stop processing if file download fails
```

Comment:
might help to give an error to the user saying that the bot couldn't download it?